.github/issue_template.md

```diff
@@ -1,8 +1,8 @@
 <!--- Remove text and sections that do not apply -->
 
-The issue tracker is only for reporting bugs or asking for feature requests and bugs.
-If you need technical assistance for running the IOTA Javascript Library please consult the #javascript channel on discord (https://discord.gg/jrxApWC) or https://forum.helloiota.com/Technology/Help.
-If you have general questions on IOTA you can go to https://iota.stackexchange.com/, or https://helloiota.com/, or browse discord channels (https://discord.gg/C88Wexg).
+The issue tracker is for reporting bugs or asking for feature requests.
+If you need technical assistance, please go to the #javascript channel on https://discord.iota.org/ or https://forum.helloiota.com/Technology/Help.
```

Comment:
Would it be better to use anchor links? E.g.,

Reply:
My other PR #464 resolves this